lua/entities/gmod_wire_thruster.lua

```diff
@@ -299,7 +304,7 @@ function ENT:OnRestore()
 
 	self.ThrustOffset 	= Vector( 0, 0, max.z )
 	self.ThrustOffsetR 	= Vector( 0, 0, min.z )
-	self.ForceAngle		= self.ThrustOffset:GetNormalized() * -1
```

Comment:
These are set already in Initialize() right? Should just remove these, also the lines above.